public/app/features/dashboard/dashgrid/PanelChrome.tsx

```diff
@@ -93,6 +103,9 @@ export class PanelChrome extends PureComponent<Props, State> {
     this.props.panel.events.off(PanelEvents.refresh, this.onRefresh);
     this.props.panel.events.off(PanelEvents.render, this.onRender);
 
+    // unsubscribe to any events the panel subscribed to
+    this.panelBus.unsubscribe();
```

Comment:
Hmm, this seemed strange in the `componentDidMount ` context I would expect this in `componentWillUnMount`

Reply:
This is in unmount function , just diff that misses that